modules/oe_theme_helper/src/Plugin/Block/PageHeaderBlock.php

```diff
@@ -110,6 +110,7 @@ public function build(): array {
       '#title' => $title,
       '#introduction' => $metadata['introduction'] ?? '',
       '#metas' => $metadata['metas'] ?? [],
+      '#infos' => $metadata['infos'] ?? [],
```

Comment:
The `infos` part was not originally supported by `@PageHeaderMetadata`, or rather: it's not handled but technically we can support it.

I suggest adding a documentation file here `./docs` where we explain how the metadata plugin works and what are the supported elements that implementations can fill in, such as `intriduction`, `metas` and `infos`.

Reply:
I created an issue https://github.com/openeuropa/oe_theme/issues/250